experimental/packages/opentelemetry-instrumentation/package.json

```diff
@@ -71,6 +71,7 @@
   },
   "dependencies": {
     "@opentelemetry/api-logs": "0.53.0",
+    "@opentelemetry/sdk-logs": "^0.53.0",
```

Comment:
Was this change intentional? I don't see any usage of the `sdk-logs` package in the example above.